XML/XSD/Objects/TargetUrl.cs

```diff
@@ -44,19 +43,6 @@ public string Url
         }
     }
     
-    [XmlAttribute]
-    public ushort Port
```

Comment:
why does this need to be a thing, vs `xs:anyURI` with a fully formed uri (e.g. `http://asdf.biz:9999/wefwef`)

Reply:
yes this is why it was removed

Reply:
god lol
![image](https://user-images.githubusercontent.com/194969/75919796-0c2eba80-5e13-11ea-921b-ef4f9efbd47d.png)